canonical/cn.ubuntu.com
# feat: Add sitemaps

## Done

- Added sitemaps
- Drive bys:
  -  Renamed `/contact/thank_you` to `/contact/thank-you` to align with our url convention and ran djlint
  -  Removed `/support/advantage` page and redirected to `/support/pro`

## QA

- Check that the following sitemaps load correctly (I based this off the u.com sitemaps so feel free to compare against those):
  - [index](https://cn-ubuntu-com-731.demos.haus/sitemap.xml)
  - [base pages](https://cn-ubuntu-com-731.demos.haus/static/files/sitemap.xml)
  - [blog](https://cn-ubuntu-com-731.demos.haus/blog/sitemap.xml)
  - [engage](https://cn-ubuntu-com-731.demos.haus/engage/sitemap.xml) 

## Issue / Card
https://warthogs.atlassian.net/browse/WD-14473
